src/lib/seam/devices/index.ts

```diff
@@ -1 +1,9 @@
+import type { LockDevice } from 'seamapi'
+
 export * from './use-devices.js'
+
+export const getLockModel = ({ properties }: LockDevice) => {
```

Comment:
```suggestion
export const getDeviceModel = ({ properties }: LockDevice): string | undefined => {
```

Reply:
Updated in https://github.com/seamapi/react/pull/124/commits/990b30a8a4a9e9a1c9418e097898b1d1c1eda2fd